### PR TITLE
Replace this repo's copy of Encoding and Decoding.lvlib with lvlib in custom device development tools

### LINKED ISSUE #174: replace owned encoding and decoding lvlib with shared lvlib
**Describe the bug**
Encoding and Decoding library was copied from this repo to https://github.com/ni/niveristand-custom-device-development-tools. Once https://github.com/ni/niveristand-custom-device-development-tools/pull/41 approved and merged. This same library should be removed from this repo and the custom device updated.

**To Reproduce**
1. N/A, this is a code reuse/design issue.

**Expected behavior**
Encoding and Decoding library should be shared by all avionics communications bus custom devices.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10 Enterprise

**Additional context**

### PR DESCRIPTION
**Pull request by dbendele**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-ballard-arinc429-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Replace local copy of Encoding and Decoding.lvlib with lvlib installed to vi.lib by veristand custom device development tools.

### Why should this Pull Request be merged?

Fixes #174 by reusing Encoding and Decoding.lvlib from custom device development tools.

### What testing has been done?

Built successfully and all unit tests pass.

### CODE REVIEW

**Review by buckd — COMMENTED** [review]
Nothing is being relinked. Are you just relying on things to be found because the are now in `vi.lib`?

- [x] `test Create and Look Into Rx Execution Unit Map.vi` asks where to load `Encoding.ctl`.
- [ ] `test Encode and Decode (BNR, signed).vi` asks where to load `BCD Decode Integer.vi`. (And likely all the tests in `Encoding and Decoding.lvclass`).

**Review by buckd — APPROVED** [review]

**Comment by dbendele** [comment]
merged main and resolved conflicts in support project; waiting on successful PR Build before squash merge

**Pull request merged**
